WORKSPACE

```diff
@@ -1,37 +1,58 @@
 workspace(name = "angular_material")
 
 # Add nodejs rules
-http_archive(
+git_repository(
```

Comment:
@alexeagle actually says that http_archive is better

Reply:
I just learned about `git_repository` from https://github.com/bazelbuild/rules_nodejs#installation. For me, this seems to be more readable and easier to maintain. But not sure about integrity